src/components/ToDoList.js

```diff
@@ -0,0 +1,44 @@
+// your components will all go in this `component` directory.
+// feel free to change this component.js into TodoList.js
+
+import React, { Component } from 'react';
+import ToDo from "./Todo"
```

Comment:
When I tried to open this app, it wouldn't render because `./Todo` should be `./toDo` Keep an eye on this.